packages/labs/cli/src/lib/lit-cli.ts

```diff
@@ -252,10 +252,19 @@ export class LitCli {
     const reference = maybeReference;
     let resolvedPackageLocation = this.resolveImportForReference(reference);
     if (resolvedPackageLocation === undefined) {
-      const installed = await this.installDepWithPermission(reference);
+      const installed = await installDepWithPermission({
+        description: `The command ${reference.name}`,
```

Comment:
This is clearer with the command name quoted, recommend JSON.stringify(reference.name)